### DISCUSSION TITLE
Ctrl+Shift for skew tool

### DISCUSSION THREAD

**Discussion by StickmanRed** [ScratchAddons/ScratchAddons#7934] [Ideas]
Pressing Shift doesn't seem to do anything yet when using the skew tool... I was thinking of assigning it to skewing in the regular direction + scaling/stretching the perpendicular. Would this be a good idea?

**Comment by WorldLanguages** [comment]
@CST1229 Any opinions?

**Reply by CST1229** [reply]
i think this is good
